docs/content/docs/connectors/datastream/jdbc.md

```diff
@@ -26,16 +26,211 @@ under the License.
 
 # JDBC Connector
 
-This connector provides a sink that writes data to a JDBC database.
+This connector provides a source that read data from a JDBC database and
+provides a sink that writes data to a JDBC database.
 
 To use it, add the following dependency to your project (along with your JDBC driver):
 
 {{< connector_artifact flink-connector-jdbc jdbc >}}
 
-Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{< ref "docs/dev/configuration/overview" >}}).
-A driver dependency is also required to connect to a specified database. Please consult your database documentation on how to add the corresponding driver.
+Note that the streaming connectors are currently __NOT__ part of the binary distribution.
+See how to link with them for cluster execution [here]({{< ref "docs/dev/configuration/overview" >}}).
+A driver dependency is also required to connect to a specified database.
+Please consult your database documentation on how to add the corresponding driver.
 
-## `JdbcSink.sink`
+## Source of JDBC Connector(Experimental)
+
+Configuration goes as follow (see also {{< javadoc file="org/apache/flink/connector/jdbc/source/JdbcSource.html" name="JdbcSource javadoc" >}}
+and {{< javadoc file="org/apache/flink/connector/jdbc/source/JdbcSourceBuilder.html" name="JdbcSourceBuilder javadoc" >}}).
+
+### `JdbcSource.builder`
```

Comment:
```suggestion
## Jdbc Source
```

Jdbc Source can be as the second level instead of the third level. You can refer Kafka connector doc[1]. `Kafka Source`, `Kafka SourceFunction` and `Kafka Producer` as the second level. We don't need to distinguish the source or sink. (Current level is a little fine-grained.) 

[1] https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/kafka/#kafka-source